src/runtime/storageNG/handle.ts

```diff
@@ -123,6 +123,7 @@ export abstract class Handle<StorageType extends CRDTTypeRecord> {
   // - notifySync (bool): if keepSynced is true, call onHandleSync when the full data is received
   // - notifyUpdate (bool): call onHandleUpdate for every change event received
   // - notifyDesync (bool): if keepSynced is true, call onHandleDesync when desync is detected
+  // If both keepSynced and notifyUpdate are true, updates are only delivered while the proxy is synchronized.
```

Comment:
I'm not sure we actually want this behaviour. Can you give some details on the motivation for this?

Reply:
As per offline conversation, this *is* probably the right thing, but I would suggest updating the comment inline:
```
// - notifyUpdate (bool): if keepSynced is true, call onHandleUpdate for every change event received if the proxy is currently synchronized (i.e. updates received when desynced are ignored); if keepSynced is false, just notify for all updates
```